recipes-bsp/u-boot/u-boot-iot2050.inc

```diff
@@ -46,6 +46,10 @@ TEMPLATE_VARS += "FIRMWARE_SECURE_VER"
 
 U_BOOT_CONFIG_PACKAGE = "1"
 
+# break dependency chain by adding support for runtime provides / depends
+# TODO: once this is globally enabled in ISAR, remove this line
+RPROVIDES += "${PROVIDES}"
+
```

Comment:
IIUC, we should wait for Isar to pick up the generic solution.

Reply:
We now have the support in ISAR. Once we have the update here in meta-iot2050 as well, I will update this.
A similar change also needs to be done in `meta-coral` to reduce the number of components that are re-build on changes.